index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="mobile.css">
+</head>
+<body>
+<div class="background">
+  
+  <header>
+    <nav>
+      <div class = "top-nav">
+        <ul class="nav-ul">
+            <li><a href="#" title="facebook"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="#" title="twitter"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+    </div>
+    <div class="bottom-nav">
+      <div class="icon">
+        <a href="index.html"><i class="fas fa-code fa-3x"></i>
+          <h1>LL Coding Together</h1>
+        </a>
+        <ul class="links">
+          <li class="link"><a href="about.htm">About</a></li>
+          <li class="link"><a href="">Program</a></li>
+          <li class="link"><a href="">Join</a></li>
+          <li class="link"><a href="">Sponsor</a></li>
+          <li class="link"><a href="">News</a></li>
+          <li class="link"><a id="border" href="">LL Coding</a></li>
+        </ul>
+      </div>
+    </div>
+
+      <i class="fas fa-bars fa-3x"></i>
+      <nav id="menu" class="invisible">
+        <ul class="menu-ul">
+            <li><i class="fal fa-times fa-3x cancel-icon"></i></li>
+            <li><a href="about.htm">About</a></li>
+            <li><a href="#About">Join</a></li>
+            <li><a href="#Contact">Sponsor</a></li>
+            <li><a href="#Contact">News</a></li>
+            <li><a href="#Contact">LL Coding</a></li>
+        </ul>
+    </nav>
+    </nav>
+  </header>
+
+  <main>
+    <section class="section-1">
+      <h2>"Hello World!"</h2>
+      <h1>LET'S LEARN CODING TOGETHER</h1>
+      <p>Platform that consists of an interactive learning web platform, an online community forum, chat rooms, online publications and local organizations that intend to make learning web development accessible to anyone. </p>
+      <article>
+        <h2>2021.10.22(FRI) ~ 30(SAT)</h2>
+        <p>@ Addis Ababa, Ethiopia, Africa.</p>
+      </article>
+
+    </section>
+  </main>
+</div>
+
```

Comment:
- [ ] There is too much white space between this two elements (This also happens in the desktop view), please make it similar to the one in the design to comply with the project requirements:

![image](https://user-images.githubusercontent.com/38522578/137224491-de758580-1e84-4b0a-8621-d2e7e1109f96.png)